config/install/migrate_plus.migration.station_nodes.yml

```diff
@@ -0,0 +1,360 @@
+  -
+    name: field_case_conference_email_link
+    label: 'Case conference email Link'
+    selector: /attributes/Case_Conf_Enq_Email_Link
+  -
+    name: field_prosecution_unit_notice
+    label: 'Prosecution Unit notice'
+    selector: /attributes/Prosecution_Unit_Notice
+  -
+    name: field_opening_hours
+    label: 'Opening hours'
+    selector: /attributes/General
+  -
+    name: field_opening_hours_notice
+    label: 'Opening hours notice'
+    selector: /attributes/Notice
+  -
+    name: field_specialty_services_or_faci_CIU
+    label: 'Specialty services or facilities'
+    selector: /attributes/Crime_Investigate_Unit_CIU
+  -
+    name: field_specialty_services_or_faci_CPO
+    label: 'Specialty services or facilities'
+    selector: /attributes/Crime_Prevent_Officer_CPO
+  -
+    name: field_specialty_services_or_faci_FVIU
+    label: 'Specialty services or facilities'
+    selector: /attributes/FVIU
+  -
+    name: field_specialty_services_or_faci_FVLO
+    label: 'Specialty services or facilities'
+    selector: /attributes/FVLO
+  -
+    name: field_specialty_services_or_faci_FCLO
+    label: 'Specialty services or facilities'
+    selector: /attributes/FCLO
+  -
+    name: field_specialty_services_or_faci_Justice_Of_The_Peace
+    label: 'Specialty services or facilities'
+    selector: /attributes/Justice_Of_The_Peace
+  -
+    name: field_specialty_services_or_faci_LGBTIQ
+    label: 'Specialty services or facilities'
+    selector: /attributes/LGBTIQ_Liaison_Officer_LLO
+  -
+    name: field_specialty_services_or_faci_NW
+    label: 'Specialty services or facilities'
+    selector: /attributes/Neighbourhood_Watch
+  -
+    name: field_specialty_services_or_faci_PPU
+    label: 'Specialty services or facilities'
+    selector: /attributes/Pro_active_Police_Unit_PPU
+  -
+    name: field_specialty_services_or_faci_PU
+    label: 'Specialty services or facilities'
+    selector: /attributes/Prosecutions_Unit
+  -
+    name: field_specialty_services_or_faci_RFO
+    label: 'Specialty services or facilities'
+    selector: /attributes/Regional_Firearms_Officers
+  -
+    name: field_specialty_services_or_faci_SOCIT
+    label: 'Specialty services or facilities'
+    selector: /attributes/SOCIT
+  -
+    name: field_specialty_services_or_faci_VASO
+    label: 'Specialty services or facilities'
+    selector: /attributes/Victim_Assist_Supp_Officer
+  -
+    name: field_specialty_services_or_faci_YRO
+    label: 'Specialty services or facilities'
+    selector: /attributes/Youth_Resource_Officer
+  -
+    name: field_accessibility
+    label: 'Accessibility'
+    selector: /attributes/Accessibility
+destination:
+  plugin: entity:node
+  default_bundle: station
+process:
+  type:
+    plugin: default_value
+    default_value: station
+  title: title
+  field_street_address: field_street_address
+  field_suburb: field_suburb
+  field_state/target_id:
+    plugin: default_value
+    default_value: 4506
```

Comment:
@g2410g Pls do not use this. 
I would be happy for you to do the migration for this vocab as well.

Reply:
Sure @gargsuchi I will do that as well.